datafusion/physical-expr/Cargo.toml

```diff
@@ -34,13 +34,16 @@ path = "src/lib.rs"
 
 [features]
 crypto_expressions = ["md-5", "sha2", "blake2", "blake3"]
-default = ["crypto_expressions", "regex_expressions", "unicode_expressions", "encoding_expressions"]
+default = ["crypto_expressions", "regex_expressions", "unicode_expressions", "encoding_expressions",
```

Comment:
this is a strange reformatting (as is the reformatting in the other .toml files) but I don't see anything wrong with it. 

If you can figure out how to avoid such changes I think it would result in smaller diffs that are easier to review

Reply:
Agreed, this kind of thing is distracting. Must be some VS code config thing, we'll share if we figure out why/how it turns up.